pkgs/games/liberal-crime-squad-git/default.nix

```diff
@@ -0,0 +1,28 @@
+{ fetchFromGitHub, stdenv, ncurses, autoconf, automake , SDL2, SDL2_mixer }:
+
+stdenv.mkDerivation rec {
+  version = "2016-03-03";
+  name = "liberal-crime-squad-git-${version}";
+  src = fetchFromGitHub {
+    owner = "Kamal-Sadek";
+    repo = "Liberal-Crime-Squad";
+    rev = "39c5f167e902e33cb37b152215d67f71b9ae8269";
+    sha256 = "18s2w2570fd79y9rb7ikxls1qw29l1lbwk9n2dxxqjslbzrbvv31";
+  };
+
+  preConfigure = ''
+    ./bootstrap
+  '';
+
+  buildInputs = [ ncurses autoconf automake SDL2 SDL2_mixer ];
```

Comment:
Maybe `autoreconfHook` (with no call to `./bootstrap`) would be cleaner, but that's of no big concern.